SpywareFilter/sections/tracking_servers.txt

```diff
@@ -42,6 +42,8 @@
 ||visilabs.com^$third-party
 ||visilabs.net^$third-party
 !
+||tracking*.adweb.co.kr^
+||wcs.naver.com^
```

Comment:
This rule will break a lot of Korean sites. As I remember, it was removed from our filter.

Reply:
```suggestion
||wcs.naver.com^$domain=danawa.com
```
Then, what about this?

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/76128/commits/4bd74e3760dfa08dcbc6f5d3f95d09a33bdd8d3e